arm-mediaservices/2015-10-01/swagger/media.json

```diff
@@ -301,29 +338,52 @@
               "$ref": "#/definitions/MediaService"
             }
           }
-        ]
-      }
-    },
-    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Media/mediaservices/{mediaServiceName}/regenerateKey": {
-      "post": {
-        "summary": "Regenerate the key for a Media Service.",
-        "operationId": "MediaService_RegenerateKey",
+        ],
         "responses": {
           "200": {
-            "description": "OK",
+            "description": "Success. The Media Services account was updated.",
             "schema": {
-              "$ref": "#/definitions/RegenerateKeyOutput"
+              "$ref": "#/definitions/MediaService"
+            },
+            "examples" : {
+              "application/json": {
+                "name": "contosomedia",
+                "id": "/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/contoso-media/providers/Microsoft.Media/mediaservices/contosomedia",
+                "type": "Microsoft.Media/mediaservices",
+                "location": "West US",
+                "tags":null,
+                "properties":{  
+                  "apiEndpoints":[  
+                    {  
+                      "endpoint": "https://wamsbayclus001rest-hs.cloudapp.net/api/",
+                      "majorVersion": "2"
+                    }
+                  ],
+                  "storageAccounts":[  
+                    {  
+                      "id": "/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/contoso-media/providers/Microsoft.Storage/storageAccounts/contosomedia",
+                      "isPrimary": true
+                    }
+                  ]
+                }  
+              }
             }
           },
-          "202": {
-            "description": "Accepted"
-          },
           "default": {
-            "description": "Detailed error information.",
+            "description": "Update Media Services failed.",

```

Comment:
Would this be better as "Update Media Services failures.", or "Update Media Services that failed", or "Update Media Services operation failed." ?